### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by ashfire908**
* Fixes make task
* Allows doc gen without having to install package
* Uses readme for doc index
* Unifies metadata
* Misc other changes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,7 +59,8 @@ To test a single test, use e.g. ``python -m unittest tests.test_client.ClientTes
 Documenting
 ~~~~~~~~~~~
 
-Run ``make docs``, then open ``docs/_build/html/index.html`` in a web browser.
+Install the ``doc`` extras for the package.
```
Comment (andy-faunadb) [inline README.rst]: A specific shell command would be better.

**Comment by andy-faunadb** [comment]
What is `docs/_static/.keep` for?

**Comment by ashfire908** [comment]
To suppress a warning by sphinx about that folder being missing.

**Comment by andy-faunadb** [comment]
LGTM

**Pull request merged**